superdesk/default_settings.py

```diff
@@ -190,7 +190,11 @@ def local_to_utc_hour(hour):
                     "char_filter": ["html_strip_filter"],
                 },
             },
-        }
+        },
+        "query_string": {
```

Comment:
seems like this config is not available in 7, and given that we set it per query, I suppose it's not needed anyhow right?

Reply:
I did not test it yet (PR is not finished). The option in the es7 docs. 
I will test it and let you know.